comms/dht/src/connectivity/mod.rs

```diff
@@ -289,7 +291,7 @@ impl DhtConnectivity {
         match event {
             DhtEvent::NetworkDiscoveryPeersAdded(info) => {
```

Comment:
We should just delay when this event is triggered. This event comes from peer sync, which is a state machine. The state machine should enter a "sleep" state for a configurable  period, then kick into peer discovery mode after that period, which will delay when the peer pools are refreshed

Reply:
Let met try that